include/beman/optional26/detail/iterator.hpp

```diff
@@ -43,18 +43,20 @@ struct contiguous_iterator : public base_contiguous_iterator<T, Container> {
     using typename base_type::reference;
 
     // Default constructor.
-    contiguous_iterator() noexcept : m_current() {}
+    constexpr contiguous_iterator() noexcept : m_current() {}
```

Comment:
I think these changes look fine.

@steve-downey @tzlaine , do you have any other comment?